src/articles/articles.view.js

```diff
@@ -0,0 +1,104 @@
+import { EventBus } from "../EventBus.js";
+import { View } from "../View.js";
+
+export class ArticlesView extends View {
+  _container = document.getElementById("blogs__list");
+  _buttonMoreArticles = document.getElementById("more-articles");
+  _containerSelected = document.getElementById("blog__preview");
+  eventTypes = {
+    moreArticlesPressed: "moreArticlesPressed",
+    articleClicked: "articleClicked",
+  };
+
+  constructor() {
+    super();
+    this.eventBus = new EventBus(this.eventTypes);
+    this._buttonMoreArticles.addEventListener("click", () =>
+      this.eventBus.notify(this.eventTypes.moreArticlesPressed),
+    );
+
+    this._container.addEventListener(
+      "click",
+      this._handleArticleClicks.bind(this),
+    );
+    this.DELAY_ITEM = 500;
+  }
+
+  subscribe(eventType, listener) {
+    this.eventBus.subscribe(eventType, listener);
+  }
+
+  async appendEnd(data) {
+    for (const article of data) {
+      const wrapper = document.createElement("div");
+      wrapper.innerHTML = this._generateItem(article).trim();
+      const item = wrapper.firstElementChild;
+
+      item.classList.add("blogs__item--invisible");
+      this._container.appendChild(item);
+      this._container.scrollIntoView({ behavior: "smooth" });
```

Comment:
Let's better call `scrollIntoView` on item instead of container 

```suggestion
     item.scrollIntoView({ behavior: "smooth" });

```